$NetBSD$

include of stdio.h required for QNX to avoid this compile error:

/usr/qnx632/target/qnx6/usr/include/stdio.h:260: error: conflicting types for `fseeko'
/usr/qnx632/target/qnx6/usr/include/stdio.h:188: error: previous declaration of `fseeko'


--- gtk/gtkdialog.c.orig	2007-09-13 22:35:40.000000000 +0200
+++ gtk/gtkdialog.c
@@ -23,7 +23,7 @@
  * files for a list of changes.  These files are distributed with
  * GTK+ at ftp://ftp.gtk.org/pub/gtk/. 
  */
-
+#include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <config.h>
